assets/scripts/configure-microk8s-windows.ps1

```diff
@@ -337,11 +322,33 @@ checkExitCode;
 # Ensure directory is safe
 #=========================
 
-Write-Host "Marking repositories as safe directories"
+Write-Host "Checking if repositories are marked as safe directories"
+
+$distro = cleanseString(wsl bash -ic 'echo $WSL_DISTRO_NAME');
+$distro = $distro.ToString().Trim();
+$isEngineSafe = $true
+$isOpsSafe = $true
+
+$engineStatusCommand = "git -C \\wsl.localhost\$distro$ENGINE_FOLDER status";
+$opsStatusCommand = "git -C \\wsl.localhost\$distro$OPS_FOLDER status";
+
+$engineOutput = Invoke-Expression "& $engineStatusCommand 2>&1";
+$opsOutput = Invoke-Expression "& $opsStatusCommand 2>&1";
 
-setSafeDirectory($ENGINE_FOLDER);
+if ([String]$engineOutput -match "dubious ownership") {
+    $isEngineSafe = $false
+}
+
+if ([String]$opsOutput -match "dubious ownership") {
+    $isOpsSafe = $false
+}
+
+if (($isEngineSafe -eq $false) -or ($isOpsSafe -eq $false)) {
+    Write-Host "Marking repositories as safe directories"
+    Start-Process powershell -PassThru -Wait -verb runas -ArgumentList "-file $PSScriptRoot\set-git-safe-directory.ps1 -e '$ENGINE_FOLDER' -o '$OPS_FOLDER' -d '$distro' -es '$isEngineSafe' -os '$isOpsSafe'"
```

Comment:
I think `'$isEngineSafe'` & '$isOpsSafe' does not needs to be sent in quotes as they are already booleans. This will unnecssarily convert them to string.